rudder-web/src/main/scala/com/normation/rudder/web/components/TechniqueEditForm.scala

```diff
@@ -210,7 +212,9 @@ class TechniqueEditForm(
         case e:EmptyBox => NodeSeq.Empty
```

Comment:
I'm not sure, the code seems to use the box as an Option. I still can log on failure.